File: 0027-remove-element/0027-remove-element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    let current = 0;
    let k = 0;
    while(current < nums.length){
        if (nums[current] != val){
            nums[k] = nums[current]
            k++
        }
       current++}
    return k
};

    
        
    
    
    
    
    
    

File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k = 0
        current = 0
        start = 0
        while start < len(nums):
            if nums[current] != val:
                nums[k] = nums[current]
                k += 1
            current += 1
            start += 1
        return k
        